### ISSUE TITLE
Crop photo for the contact picture

### ISSUE THREAD

**Issue by Spartachetto** [nextcloud/contacts#1662]
Nextcloud version: 18.0.3
Contacts verion: 3.3.0

I tried to assign a photo to a contact and I had some issues.
You can read part of them here:
https://github.com/nextcloud/nextcloud-vue/issues/631#issuecomment-642947863

This issue is about another thing. When I selected the right photo on my Nextcloud instance, I was expecting to have the possibility to crop the photo I.e. to select a part of the photo with an easy interactive tool. I was accustomed by other websites that I could select a part of the image "zooming " on the person of interest. I am aware that with that operation you lose quality (=you have a worse resolution). Yet it is very practical, otherwise you have to download the photo and create a new file with a image modification program.

**Comment by call-me-matt** [comment]
just a remark with the same context: I find it also confusing that the contact list on the left shows the top part of the picture, while the selected contact's avatar is showing the center part of the image; so you have the same picture side by side being cropped differently.

**Comment by calvinrw** [comment]
I agree with both points completely. On other platforms, a user will typically be asked to zoom and crop a photo before using it as an avatar.

I think the best solution to drop all instances of automatic cropping, beyond what is necessary to generate appropriately-sized thumbnails, in favor of allowing a user to manually edit a photo before assigning it as an avatar.

![Screenshot_2020-06-14_19-59-48](https://user-images.githubusercontent.com/687129/84608825-cc4b8a80-ae79-11ea-85d1-6bccde6fd2dc.png)

**Comment by Spartachetto** [comment]
Nextcloud version: 18.0.3

I found that this functionality is available when you pick a photo for your user profile.
Maybe you could reuse this for contacts...
